src/api/app/views/webui/request/show.html.erb

```diff
@@ -115,8 +115,8 @@
                               Forward to developed package
                               <%= project_or_package_link project: forward[:project], package: forward[:package], short: true %>
                               <br/>
-                              <% index += 1 %>
                           <% end %>
+                          <% index += 1 %>
```

Comment:
If you move this to here then can be gaps in the sequence. I mean,  you can have checkboxes for index 1, 3, 4, 8 ... I don't know if this is good or a problem. If this is not a problem, then I would delete this line and line 106 of this PR and then change the loop for an `each_with_index`